S/SLATE/build_tarballs.jl

```diff
@@ -20,18 +19,27 @@ script = raw"""
 cd slate
 git submodule update --init
 mkdir build && cd build
-cmake \
-  -DCMAKE_INSTALL_PREFIX=${prefix} \
-  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-  -DCMAKE_BUILD_TYPE="Release" \
-  -Dblas=openblas \
-  -Dbuild_tests=no \
-  -DMPI_RUN_RESULT_CXX_libver_mpi_normal="0" \
-  -DMPI_RUN_RESULT_CXX_libver_mpi_normal__TRYRUN_OUTPUT="" \
-  -Drun_result="0" \
-  -Drun_result__TRYRUN_OUTPUT="ok" \
-  -Dc_api=yes \
-  ..
+
+CMAKE_FLAGS=()
+CMAKE_FLAGS=()
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
```

Comment:
Why not
```suggestion
CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=${prefix})
```
?

Also why not a single `CMAKE_FLAGS` setting?